src/locale_common.h

```diff
@@ -288,6 +288,7 @@ MAKE_ENUM(enum_flow, FL_(off), FL_(flow), FL_(bufferedflow), FL_(buffer), FL_(la
 MAKE_ENUM(enum_reset, FL_(dash), FL_(maintenance), FL_(error), FL_(history), FL_(message))
 MAKE_ENUM(enum_maxHeat, FL_(0kW), FL_(2kW), FL_(3kW), FL_(4kW), FL_(6kW), FL_(9kW))
 MAKE_ENUM(enum_pumpMode, FL_(proportional), FL_(deltaP1), FL_(deltaP2), FL_(deltaP3), FL_(deltaP4))
+MAKE_ENUM(enum_pumpCharacter, FL_(proportional), FL_(preassure1), FL_(preassure2), FL_(preassure3), FL_(preassure4), FL_(preassure5), FL_(preassure6))
```

Comment:
From maintainers point of view should this be as I put it (`preassure1`, `preassure2`, ...) or as per manual (`preassure150`, `preassure200`, ...). My thoughts were that actual `mbar` values may be different from boiler to boiler